### ISSUE TITLE
Feature request: Value area

### ISSUE THREAD

**Issue by IHB4L** [gbzenobi/CSharp-NT8-OrderFlowKit#29]
add a function to put in a rectangle the area where a customizable percentage of volume is. Thank you for making this free.

**Comment by dolomick** [comment]
Yes, please. Should be easy for a decent coder and this thing is almost perfect!
